wordpress-mobile/WordPress-iOS
# Issue/7219 update pods on develop

Fixes #7219 

To test:
1. `pod repo update`
2. `pod install`
3. Clean, Build, Run.
4. Verify that nothing crashes or breaks. 
5. Test that Loopback, Helpshift and Hockey SDK are working as expected.

Needs review: @astralbodies
